.travis/run_travis_job.sh

```diff
@@ -53,4 +53,25 @@ else
     if [[ "${TRAVIS_JOB_NAME}" == "Linux JDK 8 CHECK_RUST=false" ]]; then
       mvn org.eluder.coveralls:coveralls-maven-plugin:report
     fi
+
+    # Test exonum_launcher_java_plugins
```

Comment:
Would it make sense to move them to a separate job?

Reply:
Python plugins need protobuf files, so they must be run after `core generate-sources`. Do you think a separate job would be better?

Reply:
I don't know, it seems to require some separate setup (python, pip, protoc) so it makes some sense. Also it would make the feedback more focused. 

Or, would it make sense to add commands to test the python plugin to the `run_all_tests.sh` script? What would it require from us in terms of keeping them green locally, given the python3 is available (regular `pip install`s?)?

Reply:
It would be better to use virtualenv for local tests - so `./run_all_tests.sh` will not affect (and won't be affected by) other python packages installed.

Reply:
I think I'd move it to a separate script, add it to run_all_tests for local tests and also add a separate job

Reply:
Good, but I think it is rather either/or: if we add it to local tests run with `run_all_tests`, it will be run by each job.

Reply:
After some consideration, safe script for local tests seems redundant - changes to plugins are rare and tests can be run manually if needed.